### ISSUE TITLE
TypeError: fit() got an unexpected keyword argument 'tensorboard'

### ISSUE THREAD

**Issue by lc1125655184** [TXin96/VoxResNet#3]
when i run train.py,I have the following questions.i had try to solve it ,but failed,i need some advice from you,thanks

Traceback (most recent call last):
  File "train.py", line 84, in <module>
    eval_train=False, tensorboard=True,tensorboard_epoch_freq=1,tensorboard_weight_histograms=True,tensorboard_graph_vis=True)
TypeError: fit() got an unexpected keyword argument 'tensorboard'

what should i do to solve it?

**Comment by TXin96** [comment]
I don't have that problem, maybe because of the out of date version.

**Comment by lc1125655184** [comment]
I would like to ask you about the version of your tensorflow, and the version of my tensorflow-gpu is 1.10.0.

**Comment by TXin96** [comment]
1.4.1, it's more than half a year ago.

**Comment by lc1125655184** [comment]
OK，thanks

**Comment by lc1125655184** [comment]
How is the data in train/label obtained?after reading data,Results are as follows .

reading data...
finished reading data.
images shape:(100, 160, 188, 128, 1) float32
labels shape:(100, 160, 188, 128, 3) int32

I copied the pack/label to the train/label.Is there a problem with this?

**Comment by TXin96** [comment]
Nope, the last dimension of label is class number(which is 3 here) after one-hot encoding. The way to obtain data is defined in input_data.py.

**Comment by lc1125655184** [comment]
After I updated the version of tensorflow, I had the following problem during training. According to the online solution, I tried to use sudo python train.py, but the following problem still occurs. I have been tossing for a long time, I hope you can give some advice, thank you very much.

[TL] Summary name VoxResNet/res6a-conv2/b_conv3d:0 is illegal; using VoxResNet/res6a-conv2/b_conv3d_0 instead.
[TL] Param name VoxResNet/decon3a/W_deconv3d:0
INFO:tensorflow:Summary name VoxResNet/decon3a/W_deconv3d:0 is illegal; using VoxResNet/decon3a/W_deconv3d_0 instead.
[TL] Summary name VoxResNet/decon3a/W_deconv3d:0 is illegal; using VoxResNet/decon3a/W_deconv3d_0 instead.
[TL] Param name VoxResNet/decon3a/b_deconv3d:0
INFO:tensorflow:Summary name VoxResNet/decon3a/b_deconv3d:0 is illegal; using VoxResNet/decon3a/b_deconv3d_0 instead.
[TL] Summary name VoxResNet/decon3a/b_deconv3d:0 is illegal; using VoxResNet/decon3a/b_deconv3d_0 instead.
[TL] Param name VoxResNet/classifier3a/W_conv3d:0
INFO:tensorflow:Summary name VoxResNet/classifier3a/W_conv3d:0 is illegal; using VoxResNet/classifier3a/W_conv3d_0 instead.
[TL] Summary name VoxResNet/classifier3a/W_conv3d:0 is illegal; using VoxResNet/classifier3a/W_conv3d_0 instead.
[TL] Param name VoxResNet/classifier3a/b_conv3d:0
INFO:tensorflow:Summary name VoxResNet/classifier3a/b_conv3d:0 is illegal; using VoxResNet/classifier3a/b_conv3d_0 instead.
[TL] Summary name VoxResNet/classifier3a/b_conv3d:0 is illegal; using VoxResNet/classifier3a/b_conv3d_0 instead.
[TL] Param name sub:0
INFO:tensorflow:Summary name sub:0 is illegal; using sub_0 instead.
[TL] Summary name sub:0 is illegal; using sub_0 instead.
[TL] Param name sub_1:0
INFO:tensorflow:Summary name sub_1:0 is illegal; using sub_1_0 instead.
[TL] Summary name sub_1:0 is illegal; using sub_1_0 instead.
[TL] Param name sub_2:0
INFO:tensorflow:Summary name sub_2:0 is illegal; using sub_2_0 instead.
[TL] Summary name sub_2:0 is illegal; using sub_2_0 instead.
[TL] Param name sub_3:0
INFO:tensorflow:Summary name sub_3:0 is illegal; using sub_3_0 instead.
[TL] Summary name sub_3:0 is illegal; using sub_3_0 instead.
[TL] Finished! use $tensorboard --logdir=logs/ to start server
[TL] Start training the network ...
2018-10-10 11:13:53.517006: W tensorflow/core/framework/allocator.cc:101] Allocation of 1509949440 exceeds 10% of system memory.
2018-10-10 11:13:55.242556: W tensorflow/core/framework/allocator.cc:101] Allocation of 1509949440 exceeds 10% of system memory.
2018-10-10 11:13:55.737789: W tensorflow/core/framework/allocator.cc:101] Allocation of 1509949440 exceeds 10% of system memory.
2018-10-10 11:13:56.389206: W tensorflow/core/framework/allocator.cc:101] Allocation of 1509949440 exceeds 10% of system memory.
terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_alloc
已放弃

**Comment by txin96** [comment]
It seems like the memory overflow. You can try cropping the image or something else. In the paper, the overlap-tiling strategy is used to avoid memory overflow, but it's not implemented in this work.

**Comment by lc1125655184** [comment]
thanks
